AndroidSDKTests/src/test/java/com/leanplum/internal/RequestTest.java

```diff
@@ -67,6 +68,20 @@ public void setUp() {
     Leanplum.setApplicationContext(context);
   }
 
+  /**
+   * Test that read writes happened sequentially when calling sendNow().
+   */
+  @Test
+  public void testSequenceOfDatabaseOperations() {
```

Comment:
test style: tests are easier to understand and maintain when the test name is a complete sentence, in this case something like this:

public void shouldWriteRequestAndSendInSequence()